snippet/jestSetup.js

```diff
@@ -0,0 +1,9 @@
+const sinon = require("sinon");
```

Comment:
Can you move this file into our __tests__ folder?

also, can you rename `__tests__` > `test` ?